### DISCUSSION TITLE
1 bug and 1 suggestion for improvement

### DISCUSSION THREAD

**Discussion by nahuakang** [OpenMined/courses#36] [Feedback] [answered]
The bug: Every time I select a chunk of the text that belongs to a quiz region (such as one of the choices) and attempts to press `Ctrl + C`, the webpage crashes.

The suggestion for improvement: Could we somehow store progress a student has on a YouTube video? When I encountered a crash, I had to look back to where the video lesson stopped as the video always restarts from the beginning.

Cheers :rocket:

**Comment by mcleonard** [comment, marked as answer]
Thanks Nahua! Filing this for future work 👍 

@nahuakang Could you help us out by creating an issue here: https://github.com/OpenMined/openmined/issues
